### PR TITLE
Home page 

### PR DESCRIPTION
**Pull request by Jia-A**
In this PR review, I created, context and used APIs to get the videos and categories from my DB to my homepage.

### CODE REVIEW

File: src/pages/homepage.jsx [hunk 1]
```diff
@@ -3,9 +3,14 @@ import { Sidebar } from "../components/sidebar";
...
 import "../root.css"
 import { useState } from "react";
+import { useVideo } from "../context/videoContext";
 const Homepage = () =>{
     const [sidebar, setSidebar] = useState(true);
     const [ drop, setDrop ] = useState(false);
+
+    const { videoState } = useVideo();
+    const { videos, categories } = videoState;
+    console.log( videos, categories )
```
Comment (prachi-sahani) [inline src/pages/homepage.jsx:13]: can be removed

**Review by prachi-sahani — COMMENTED** [review]
Overall the code looks fine
Good to merge
Suggestion - consider formatting your code

File: src/pages/homepage.jsx (hunk 1 above)
Comment (Jia-A) [reply src/pages/homepage.jsx:13]: Okay I will remove it...

**Review by ShreyashMahajan — APPROVED** [review]
code is well structured and naming is easily understandable. Good to merge

**Pull request merged**